src/main/java/uk/gov/ons/census/casesvc/utility/AddressModificationValidator.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ons.census.casesvc.utility;
+
+import java.util.Set;
+import org.springframework.util.StringUtils;
+import uk.gov.ons.census.casesvc.model.dto.ModifiedAddress;
+
+public class AddressModificationValidator {
```

Comment:
hmmm. I would have done this as a `@Component` and had the `estabTypes` injected into it by Spring

Reply:
but then I suppose we'd have to inject the `AddressModificationValidator` into the two places it's needed.

Hmmm. Disregard if you want